api/v1/models.go

```diff
@@ -131,32 +131,33 @@ func LayerFromDatabaseModel(db database.Datastore, dbLayer database.Layer, linea
 
 	notes := getNotes(layer.NamespaceName, uncertifiedRHEL)
 
-	if (withFeatures || withVulnerabilities) && (dbLayer.Features != nil || namespaces.IsRHELNamespace(layer.NamespaceName)) {
-		for _, dbFeatureVersion := range dbLayer.Features {
-			feature := featureFromDatabaseModel(dbFeatureVersion, opts.GetUncertifiedRHEL(), depMap)
+	if withFeatures || withVulnerabilities {
+		if dbLayer.Features != nil || namespaces.IsRHELNamespace(layer.NamespaceName) {
```

Comment:
would need to see if our scanning notes accounts for us still finding language vulns without OS-level vulns. Don't think we considered this possibility before

Reply:
In the [GetComponents](https://github.com/stackrox/scanner/blob/7d161d3089a7c3477cfd137ce6cd8aa180bf5a03/api/v1/models.go#L225-L227) flow used via GRPC looks like the OS-level vulns vs. language vulns are separated - FWIW